docs/source/ko/_toctree.yml

```
@@ -70,83 +71,114 @@
         title: 제로샷(zero-shot) 이미지 분류
       - local: tasks/monocular_depth_estimation
         title: 단일 영상 기반 깊이 추정
+      - local: tasks/image_to_image
```

It looks like the following task docs don't exist either so I think what you can do is replace `local: tasks/image_to_image` with `local: in_translation`. Or you could also just remove them from the toctree for now until they get translated.

Same for the `hf_quantizer` and `llm_tutorial_optimization` docs

```
- tasks/image_to_image
- tasks/image_feature_extraction
- tasks/mask_generation
- tasks/knowledge_distillation_for_image_classification
- tasks/text-to-speech
```

```suggestion
      - local: in_translation
```

---

Thanks for the review. I apologize for raising the PR without checking it properly. I'll fix the things you mentioned.